docs/src/main/asciidoc/config.adoc

```diff
@@ -625,6 +625,11 @@ quarkus:
     password: quarkus
 ----
 
+[NOTE]
+====
+Quarkus also supports using `application.yml` as the name of YAML file. The same rules apply for this file as for `application.yaml`.
```

Comment:
```suggestion
Quarkus also supports using `application.yml` as the name of the YAML file. The same rules apply for this file as for `application.yaml`.
```

Reply:
Good one! A classic mistake I always make :). You can probably spot those with closed eyes now :P